packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -418,11 +425,26 @@ function WorklistDataGrid(props) {
         id={id}
         ariaLabel={ariaLabel}
         ariaLabelledBy={ariaLabelledBy}
-        rows={rows}
+        isStriped={isStriped}
+        rows={rows.map((row) => ({
+          ...row,
+          cells: row.cells.map(cell => ({
+            ...cell,
+            isSelectable: cell.isSelectable !== false,
+          })),
+        }))}
         rowHeight={rowHeight}
         rowHeaderIndex={rowHeaderIndex}
-        pinnedColumns={pinnedColumns}
-        overflowColumns={overflowColumns}
+        pinnedColumns={pinnedColumns.map((columns) => ({
+          ...columns,
+          isSelectable: columns.isSelectable !== false,
+          isResizable: columns.isResizable !== false,
+        }))}
+        overflowColumns={overflowColumns.map((columns) => ({
+          ...columns,
+          isSelectable: columns.isSelectable !== false,
+          isResizable: columns.isResizable !== false,
```

Comment:
Looks like we are duplicating the same logic for columns.  Maybe it should be a function.